platform/solr/solr-factory-impl/src/main/java/org/codice/solr/factory/impl/Configsets.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.solr.factory.impl;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Configsets {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(Configsets.class);
+
+  protected static final List<String> SOLR_CONFIG_FILES =
+      Collections.unmodifiableList(
```

Comment:
✏️ I think ImmutableList.of would work here as well

Reply:
They would both work but we do not need the defensive copy and it looks like we do not currently use Guava directly in the factory implementations currently.  Given the issues we have had with Guava version upgrades, I would rather keep the Guava free version for now.